### DISCUSSION TITLE
QuickAccessMenuItem => larger icon needed

### DISCUSSION THREAD

**Discussion by GinoDerycke** [fluentribbon/Fluent.Ribbon#1008] [Q&A] [answered]
Hello,

is there a possibility to make the icon of a QuickAccessMenuItem larger? 
I already have set the TitleBarHeight to "36" and QuickAccessToolBarHeight to "32", but when I add QuickAccessMenuItems, the icon is still small.

My code for a QuickAccessMenuItem:

            <Fluent:QuickAccessMenuItem IsChecked="True">
                <Fluent:Button Header="{Binding Path=Ribbon_Save, Source={StaticResource Resources}}"
                               Command="{Binding SaveCommand}"
                               Icon="../Themes/Common/32x32/save.png"/>
            </Fluent:QuickAccessMenuItem>

Thanks in advance!

Gino

**Comment by batzen** [comment, marked as answer]
There is currently now built in way to do that.
To achieve what you want you would have to set `RibbonProperties.IconSize`, `Width` and `Height` on the items contained in the QAT manually. Maybe by reacting to item added events or something like that.
